### DISCUSSION TITLE
关于headerDepth不能正常使用的问题

### DISCUSSION THREAD

**Discussion by say-code** [vuepress-theme-hope/vuepress-theme-hope#3350] [Q&A] [answered]
根据文档所说，headerDepth受到 markdown.headers.level的限制，我尝试编写以下代码修改配置
const headersPluginOptions: HeadersPluginOptions= {
  level: [2,5],
  shouldAllowNested: true
}

export default defineUserConfig({

  markdown: {
    headers: headersPluginOptions

  }

之后配置  headerDepth: 5,然而依旧没有生效
我再次尝试配置  headerDepth: 1是可以生效的，
我个人感觉是level的相关配置没有生效？
大佬可以解决一下啊这个问题嘛

**Comment by Mister-Hope** [comment, marked as answer]
你把2345都加进去，你再试试

**Reply by say-code** [reply]
谢谢！已经解决了，原来这里传的数组，不是区间。
